spec/lib/video_info/providers/youtube_spec.rb

```diff
@@ -487,4 +488,36 @@
       it { is_expected.to eq 'Deafheaven - Sunbather' }
     end
   end
+
+  context "with valid video and connection timeout" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.